lib/CampMinder/SignedRequestFactory.rb

```diff
@@ -12,38 +12,21 @@ def initialize(secret_code)
   end
 
   def is_valid_request?(signed_request)
-    encoded_signature, encoded_payload = signed_request.split('.')
+    encoded_signature, encoded_payload = signed_request.split(".")
     encoded_signature === encode_signature(encoded_payload)
   end
 
   def get_payload(signed_payload)
-    Base64.decode64(prepare_decode_base64(signed_payload.split('.').last))
+    Base64.urlsafe_decode64(signed_payload.split(".").last)
   end
 
   def encode_signature(encoded_payload)
-    prepare_encoded_base64(Base64.strict_encode64(OpenSSL::HMAC.digest(OpenSSL::Digest.new('sha256'), @secret_code, encoded_payload))).strip
+    Base64.urlsafe_encode64(OpenSSL::HMAC.digest(OpenSSL::Digest.new("sha256"), @secret_code, encoded_payload)).strip()
```

Comment:
Line is too long. [119/80]<br>Do not use parentheses for method calls with no arguments.